Simplenote/SPCardPresentationController.swift

```diff
@@ -131,22 +136,26 @@ private extension SPCardPresentationController {
         cardView.translatesAutoresizingMaskIntoConstraints = false
         containerView.addSubview(cardView)
 
-        compactLayoutConstraints = [
+        compactWidthLayoutConstraints = [
             cardView.leadingAnchor.constraint(equalTo: containerView.safeAreaLayoutGuide.leadingAnchor),
             cardView.trailingAnchor.constraint(equalTo: containerView.safeAreaLayoutGuide.trailingAnchor),
         ]
 
-        regularLayoutConstraints = [
+        regularWidthLayoutConstraints = [
             cardView.leadingAnchor.constraint(equalTo: containerView.readableContentGuide.leadingAnchor),
             cardView.trailingAnchor.constraint(equalTo: containerView.readableContentGuide.trailingAnchor),
         ]
 
+        regularHeightLayoutConstraints = [
+            cardView.topAnchor.constraint(greaterThanOrEqualTo: containerView.centerYAnchor),
+        ]
+
         let heightConstraint = cardView.heightAnchor.constraint(equalToConstant: 0)
         heightConstraint.priority = .fittingSizeLevel
 
         NSLayoutConstraint.activate([
             cardView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
-            cardView.topAnchor.constraint(greaterThanOrEqualTo: containerView.safeAreaLayoutGuide.topAnchor),
+            cardView.topAnchor.constraint(greaterThanOrEqualTo: containerView.safeAreaLayoutGuide.topAnchor, constant: 10),
```

Comment:
Nipticky: Maybe move the `10` to a constant?

Reply:
Good point, will do!